manageiq-operator/api/v1alpha1/helpers/miq-components/kafka/kafka.go

```diff
@@ -0,0 +1,459 @@
+package miqkafka
+
+import (
+	"bytes"
+	"context"
+	miqv1alpha1 "github.com/ManageIQ/manageiq-pods/manageiq-operator/api/v1alpha1"
+	miqtool "github.com/ManageIQ/manageiq-pods/manageiq-operator/api/v1alpha1/helpers/miq-components"
+	miqutilsv1alpha1 "github.com/ManageIQ/manageiq-pods/manageiq-operator/api/v1alpha1/miqutils"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"strconv"
+)
+
+var caGen = 0
```

Comment:
Do we need to store this somewhere?  Every time the operator boots, we start at 0 again, right?

Reply:
If the operator is redeployed then yes, we start at 0 again which I think is fine because the certs will be cleaned up and redeployed again? Let me know if that is okay.  The intent behind this global var is so that the value persists between reconciles, so if a cert needs to be renewed we can increment the previous value

Reply:
Well, the operator could be rescheduled for any reason (host going down for maintenance, resource usage, etc), so every time a new one starts, it will start at 0 and copying the certs causing kafka to be restarted could be a problem